products/sle/main.pm

```diff
@@ -759,6 +759,7 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/install_package";
         loadtest "virt_autotest/update_package";
         loadtest "virt_autotest/reboot_and_wait_up_normal";
+        loadtest "virt_autotest/download_guest_assets" if (get_var("SKIP_GUEST_INSTALL") && check_var('ARCH', 'x86_64'));
```

Comment:
If you don't mind, can you use [is_x86_64](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/Utils/Architectures.pm) here instead?

Reply:
thanks. use this package.